index.js

```diff
@@ -61,8 +61,8 @@ JSONMLParser.prototype._createSourceOptions = function() {
     };
 };
 
-module.exports = function(markup, callback) {
-    var parser = new JSONMLParser();
+module.exports = function(markup, callback, options) {
+    var parser = new JSONMLParser(options || {});
     if (isEmpty(arguments)) {
```

Comment:
This conditional needs to be changed to also work when only `options` provided, i.e. `parser()` and `parser(options: Object)` both need to return a parse stream, while `parser(markup: String, cb: Function)` and `parser(markup: String, cb: Function, options: Object)` returns `undefined` and invokes `cb` when done.